cfme/containers/provider/__init__.py

```diff
@@ -422,6 +431,23 @@ def step(self):
         self.prerequisite_view.monitor.item_select('Ad hoc Metrics')
 
 
+class ContainerObjectAllBaseView(BaseLoggedInPage):
+    """Base class for container object All view"""
+    TITLE_TEXT = ''  # Should be defined in the inherit object
```

Comment:
**REQUIRED**: Leaving this blank will mean that we will never catch an exception in the case that someone forgets to add it to the child, as the ```am_i_here/is_displayed``` will just always return ```False```. Suggestion to remove it to let things fail hard if they don't add it.